ASI Documentation: Filtering (#634)
magento/merchdocs: src/cms/media-gallery-asset-management.md

    ![]({% link images/images/enhanced-media-gallery/delete-asset-confirmation.png %}){: .zoom}
    _Delete Confirmation_

## Assets filtering

1. On the _Admin_ sidebar, go to **Content** > _Media_ > **Media Gallery**.

> 1. Click <span class="btn">Filters</span>.

Reviewer: In the screen cap this looks like a tab and not a button.

```suggestion
1. Click the **Filters** tab.
```

    ![]({% link images/images/enhanced-media-gallery/filters.png %}){: .zoom}
    _Filters_

1. It is possible to filter assets by the entities (Products, Categories, Pages, Blocks, Store Views) where the asset is being used as well as by Content Status. For example, we can easily find assets that are used in the specific category that is enabled.

>     

Reviewer: ```suggestion
    
1. Click <span class="btn">Apply Filters</span> to see the results.

   The filtering in the following example finds assets that are used in a specific category (`cars`) and are enabled.  

```

    ![]({% link images/images/enhanced-media-gallery/filtered-by-category.png %}){: .zoom}

>     _Filter by Category_

Reviewer: ```suggestion
    _Filter for Enabled Assets by Category_
```

## Finding image duplicates

> 1. Click <span class="btn">Filters</span> and select <span class="btn">Show duplicates</span> checkbox.

Reviewer: ```suggestion
1. Click the **Filters** tab and select the **Show duplicates** checkbox.
```
